PyStemmusScope/global_data/global_data_selection.py

```diff
@@ -100,7 +100,13 @@ def collect_datasets(
     data["latitude"] = latlon[0]
     data["longitude"] = latlon[1]
 
-    # TODO: Add land cover data retrieval.
-    data["IGBP_veg_long"] = "Evergreen Needleleaf Forests"
+    landcover_data = gd.cci_landcover.retrieve_landcover_data(
+        global_data_dir,
+        latlon,
+        time_range,
+        timestep,
+    )
+    data["IGBP_veg_long"] = landcover_data["IGBP_veg_long"][0]
```

Comment:
Let's get `landcover_data["IGBP_veg_long"]` as a np.array instead of a dictionary. See my comment above.